docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version '3':
+
+services:
+  application:
+    container_name: template-node-ts
+    image: node:alpine
```

Comment:
In this part I think you can use the already existing Dockerfile to build a local image instead of downloading a new one from the registry